units/host_termination.go

```diff
@@ -180,9 +181,29 @@ func (j *hostTerminationJob) Run(ctx context.Context) {
 		} else {
 			return
 		}
+	} else {
+		// Consider if the host is in between running a single-host task group
+		if j.host.LastGroup != "" {
```

Comment:
It looks like this only gets set in `ClearRunningAndSetLastTask(`, so I'm assuming this is to avoid that situation we had yesterday in evergreen-users.

Reply:
It's actually to avoid a build failure with task groups, not user hosts. This is set for EndTask for every task. https://github.com/evergreen-ci/evergreen/blob/763cae970ea8b113126184a5916c5124859fc581/model/host/host.go#L1154